src/Compilers/CSharp/Portable/Symbols/Source/SourceTypeParameterSymbol.cs

```diff
@@ -171,12 +177,15 @@ internal override CustomAttributesBag<CSharpAttributeData> GetAttributesBag()
                 var sourceMethod = this.ContainingSymbol as SourceMemberMethodSymbol;
                 if ((object)sourceMethod == null || (object)sourceMethod.SourcePartialDefinition == null)
                 {
-                    lazyAttributesStored = LoadAndValidateAttributes(OneOrMany.Create(this.MergedAttributeDeclarationSyntaxLists), ref _lazyCustomAttributesBag);
+                    lazyAttributesStored = LoadAndValidateAttributes(
```

Comment:
This is a method body diagnostics, it shouldn't go into the declaration diagnostics. We might indeed  have a race here if it is possible to get symbol (perhaps through SemanticModel) before binding the Local Function statement and then look at attributes off of symbols.